### ISSUE TITLE
How much does the loss value finally end?

### ISSUE THREAD

**Issue by huangfaan** [eladrich/pixel2style2pixel#170]
self.parser.add_argument('--output_size', default=256, type=int, help='Output size of generator')

		self.parser.add_argument('--batch_size', default=1, type=int, help='Batch size for training')
		self.parser.add_argument('--test_batch_size', default=1, type=int, help='Batch size for testing and inference')
		self.parser.add_argument('--workers', default=1, type=int, help='Number of train dataloader workers')
		self.parser.add_argument('--test_workers', default=1, type=int, help='Number of test/inference dataloader workers')

		self.parser.add_argument('--learning_rate', default=0.0001, type=float, help='Optimizer learning rate')
		self.parser.add_argument('--optim_name', default='ranger', type=str, help='Which optimizer to use')
		self.parser.add_argument('--train_decoder', default=False, type=bool, help='Whether to train the decoder model')
		self.parser.add_argument('--start_from_latent_avg', action='store_true', help='Whether to add average latent vector to generate codes from encoder.')
		self.parser.add_argument('--learn_in_w', action='store_true', help='Whether to learn in w space instead of w+')

		self.parser.add_argument('--lpips_lambda', default=0.8, type=float, help='LPIPS loss multiplier factor')
		self.parser.add_argument('--id_lambda', default=0.1, type=float, help='ID loss multiplier factor')
		self.parser.add_argument('--l2_lambda', default=1.0, type=float, help='L2 loss multiplier factor')
		self.parser.add_argument('--w_norm_lambda', default=0, type=float, help='W-norm loss multiplier factor')
		self.parser.add_argument('--lpips_lambda_crop', default=0, type=float, help='LPIPS loss multiplier factor for inner image region')
		self.parser.add_argument('--l2_lambda_crop', default=0, type=float, help='L2 loss multiplier factor for inner image region')
		self.parser.add_argument('--moco_lambda', default=0, type=float, help='Moco-based feature similarity loss multiplier factor')

Thank you for your excellent job!! I don't know when the training should end, when I train the pSp encoder on ffhq(256*256), How much does the loss value finally end.  Now：

	loss_id =  0.672258734703064
	id_improve =  -0.6722588539123535
	loss_l2 =  0.0666063129901886
	loss_lpips =  0.2247311919927597
	loss =  0.31361714005470276

**Comment by yuval-alaluf** [comment]
There is no final loss value that you should aim for. You can see when your losses converge and then compute metrics using the scripts provided in the repo. You can then compare your results with those in the paper.

**Comment by huangfaan** [comment]
ok thanks
